plugins/jobs/src/js/jobsRunNow.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-unused-vars */
+import React from "react";
+/* eslint-enable no-unused-vars */
+
+import "rxjs/add/operator/take";
+
+import { graphqlObservable } from "data-service";
+import gql from "graphql-tag";
+
+import defaultSchema from "./data/JobModel";
+
+const runNowJobMutation = gql`
+  mutation {
+    runJob(id: $jobId) {
+      jobId
+    }
+  }
+`;
+
+function jobsRunNowGraphql(jobId) {
+  return graphqlObservable(runNowJobMutation, defaultSchema, { jobId });
+}
+
+export default jobId => {
+  return {
+    label: "Run Now",
+    onItemSelect() {
+      // take(1) makes sure the observable is going to complete after finishing
+      // the request, so we don't have to care about unsubscribing.
+      jobsRunNowGraphql(jobId)
```

Comment:
You mean the import of the take? I don't really get this one.

Reply:
I mean at least `take`, but maybe `subscribe` as well. Does it make sense?

Reply:
I would find a little weird to have this import in this scope function instead of the top level. I know that there might be some build size benefits, but I would not go there.

Also for the subscribe, there is no import for it. It comes when you import Observable, and I believe you don't ever have to import it.

Reply:
I don't mean import at all:
```js
function jobsRunNowGraphql(jobId) {
  return graphqlObservable(runNowJobMutation, defaultSchema, { jobId }).take(1).subscribe();
}
```

Reply:
Now I get you then.